demo/react/README.md

```diff
@@ -73,13 +77,14 @@ Try a phrase that is in the context:
 
 The results will appear on screen:
 
-````json
+```json
```

Comment:
This last part of the `Try Rhino` section doesn't make sense anymore. Edit it to just indicate that the user should follow the context at the bottom of the screen